updated stm32_draw_demo for stm32f7

KNguyen5256 opened the pull request:
added name to alire manifest as contributor

added openocd command for stm32f7 in README

fixed drawing mode for stm32f7

added ability to switch modes with touch points

KNguyen5256 committed:
updated stm32_draw_demo for stm32f7

added name to alire manifest as contributor

added openocd command for stm32f7 in README

fixed drawing mode for stm32f7

added ability to switch modes with touch points

ohenley commented on README.md:
````diff
@@ -59,6 +59,10 @@ gnatstudio stm32_draw_demo.gpr
 openocd -f /usr/share/openocd/scripts/board/stm32f429disc1.cfg -c 'program bin/stm32_draw_demo verify reset exit'
 ```   
 
+```console
````
You kept the other cmd. You need to keep on or the other.

ohenley commented on src/stm32_draw_demo.adb:
```diff
@@ -88,95 +90,122 @@ procedure Stm32_Draw_Demo is
+            Display.Hidden_Buffer (1).Set_Source (HAL.Bitmap.Green);
+
+            declare
+               --State : constant TP_State := Touch_Panel.Get_All_Touch_Points;
```
remove commented code

ohenley commented on src/stm32_draw_demo.adb:
```diff
@@ -88,95 +90,122 @@ procedure Stm32_Draw_Demo is
+               if State'Length = 0 then
+                  Last_X := -1;
+                  Last_Y := -1;
+
```
remove unnecessary spaces

ohenley commented on src/stm32_draw_demo.adb:
```diff
@@ -88,95 +90,122 @@ procedure Stm32_Draw_Demo is
+                  Curr_X := State (State'First).X;
+                  Curr_Y := State (State'First).Y;
+
+                  --  Lines can be drawn between two consecutive points only when
```
remove commented code

ohenley reviewed (changes requested):
Thanks for the PR Kien. Please fix review.

KNguyen5256 committed:
updated README and stm32_draw_demo.adb

removed openocd command for stm32f7

removed extra comments and unnecessary code

reformatted spacing of a few code lines

ohenley reviewed (approved):
Thx!
